app/src/androidTest/java/de/test/antennapod/EspressoTestUtils.java

```diff
@@ -116,38 +116,38 @@ public void perform(UiController uiController, View view) {
      * Clear all app databases
      */
     public static void clearPreferences() {
-        File root = InstrumentationRegistry.getTargetContext().getFilesDir().getParentFile();
+        File root = InstrumentationRegistry.getInstrumentation().getTargetContext().getFilesDir().getParentFile();
         String[] sharedPreferencesFileNames = new File(root, "shared_prefs").list();
         for (String fileName : sharedPreferencesFileNames) {
             System.out.println("Cleared database: " + fileName);
-            InstrumentationRegistry.getTargetContext().getSharedPreferences(
+            InstrumentationRegistry.getInstrumentation().getTargetContext().getSharedPreferences(
                     fileName.replace(".xml", ""), Context.MODE_PRIVATE).edit().clear().commit();
         }
 
-        InstrumentationRegistry.getTargetContext().getSharedPreferences(MainActivity.PREF_NAME, Context.MODE_PRIVATE)
+        InstrumentationRegistry.getInstrumentation().getTargetContext().getSharedPreferences(MainActivity.PREF_NAME, Context.MODE_PRIVATE)
```

Comment:
I think the lines got too long here. You can view details on the CI output.

Reply:
Yeah, I'll fix those.